english/normal-dungeon/0/0-1A.md

```diff
@@ -1,7 +1,8 @@
-Your muscle is 💪stronger 
+Your muscles have gotten stronger! 💪stronger 
```

Comment:
Stronger repeated twice

Reply:
Ah yes Totally right; don't know how I missed that. I'll make the edit now!

Reply:
Change has been made!